charts/kubernetes-agent/values.yaml

```diff
@@ -9,12 +9,12 @@ imagePullSecrets: []
 #   - name: "registry-secret-name"
 
 agent:
+  # -- The name of the agent
+  # @section -- Agent values
+  name: ""
```

Comment:
The current thinking is that the agent will have a single name field which will be used for the Octopus Server name of either the worker or the target. This is also what gets used to determine the kubernetes namespace etc. This means that should the user want to install an agent worker using the same name as an existing agent target, then there will be a namespace conflict in their cluster we simply give a helm command the way things are at the moment.

Would like to hear some thoughts about this too. There are several different ways this problem can be solved..